data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -49,45 +56,177 @@ def prepare_data(self, protocol_name: str, path: str = LOCAL_STORAGE_PATH) -> di
 
-    def _calculate_liquidable_debt(self, data: dict = None):
+    @staticmethod
+    def _get_all_max_token_values(data: dict = None) -> dict:
+        """
+        Returns a dict of all token max values.
+        :param data: A dict of data.
+        :return: A dict of all token max values.
+        """
+        max_token_prices = dict()
+
+        for index, row in data.items():
+            for token, price in row[DEBT_FIELD_NAME].items():
+                if not max_token_prices.get(token):
+                    max_token_prices.update({token: {
+                        ROW_ID_FIELD_NAME: index,
+                        PRICE_FIELD_NAME: Decimal(price)
+                    }})
+
+                    continue
+
+                if price > max_token_prices[token][PRICE_FIELD_NAME]:
+                    max_token_prices[token][ROW_ID_FIELD_NAME] = index
+                    max_token_prices[token][PRICE_FIELD_NAME] = price
+
+                    continue
+
+        return max_token_prices
+
+    def _sort_by_token_pair_correspondence(
+            self,
+            parsed_data: dict[int, dict[str, str | Decimal | dict[str, Decimal]]] = None,
+            max_token_prices: dict[str, dict[str, Decimal | int]] = None
+    ) -> dict:
+        """
+        Sorts a given data by token pair correspondence.
+        Args: parsed_data( dict[
+            int - Row number,
+            dict[
+                str - Field name,
+                str - Token value
+                | Decimal - Field value (the price as example)
+                | dict[
+                    str - Token name,
+                    Decimal - Token value
+                ]
+            ]
+        ])
+        :Args: max_token_prices( dict[
+            str - Token name,
+            dict[
+                str - Field name,
+                Decimal - Token price
+                | int - row id
+                ]
+            ])
+        :return: A dictionary of a sorted data.
+        """
+        result = deepcopy(max_token_prices)
+        for debt_token, token_info in max_token_prices.items():
+            for pair in TOKEN_PAIRS:
+                if debt_token == pair[1]:
+                    collateral_token_index = 0
+                    collateral_keys = parsed_data[token_info[ROW_ID_FIELD_NAME]][
+                        COLLATERAL_FIELD_NAME
+                    ].keys()
+
+                    if pair[collateral_token_index] not in collateral_keys:
+                        if result.get(debt_token) and debt_token not in self.token_pairs.keys():
+                            del result[debt_token]
+                            break
+
+                        continue
+
+                    self.token_pairs.update({debt_token: pair[collateral_token_index]})
+
+        return result
+
+    def _calculate_liquidable_debt(
+            self, data: dict[int, dict[str, str | Decimal | dict[str, Decimal]]] = None,
```

Comment:
In my opinion, arg's annotation should be shorter. if you want to get more info, add it rather to docstring.

It makes your code more readable.
func name and arg should be short and clear.
Extra information should be part of docstring

Reply:
Done